plugins/pixart-rf/fu-pxi-firmware.c

```diff
@@ -234,3 +276,9 @@ fu_pxi_firmware_new(void)
 {
 	return FU_FIRMWARE(g_object_new(FU_TYPE_PXI_FIRMWARE, NULL));
 }
+
+gboolean
+fu_pxi_firmware_get_is_hpac_flag(FuPxiFirmware *self)
```

Comment:
just a nit; how about rename this to `fu_pxi_firmware_is_hpac`

Reply:
Revised in d60a6699e24f22fbb4b95a07d466b26a0ab63d3a